README.md

```diff
@@ -1,3 +1,5 @@
+[![Build Status](https://travis-ci.org/Grummfy/PAGI.svg)](https://travis-ci.org/Grummfy/PAGI)
```

Comment:
Will this still work when the PR is merged, or will the build then become available at https://travis-ci.org/marcelog/PAGI?

If so, please change this line.